app/components/status/SystemStatusIcon.tsx

```diff
@@ -7,6 +9,19 @@ export interface SystemStatusIconProps {
     systemStatus?: ClusterServiceStatus;
 }
 
-export default function SystemStatusIcon({ systemStatus }: SystemStatusIconProps) {
+function SystemStatusIcon({ systemStatus }: SystemStatusIconProps) {
     return <ClusterStatusIcon status={systemStatus} />;
 }
+const mapStateToProps = (state: ReduxState) => {
+    const systemStatus = state.manager.clusterStatus.status;
+    return {
+        systemStatus,
+        maintenanceStatus: state.manager.maintenance
+    };
+};
+
+const mapDispatchToProps = () => {
+    return {};
+};
```

Comment:
Probably we can shorten it to:
```suggestion
const mapDispatchToProps = () => {};
```